packages/@aws-cdk/aws-batch/test/integ.batch.expected.json

```diff
@@ -1317,72 +1317,6 @@
         "vpcVPCGW7984C166"
       ]
     },
-    "batchfargatecomputeenvResourceServiceInstanceRole94D7AA5F": {
```

Comment:
Have you actually deployed the updated integration test, and then submitted a job to the changed compute environments to make sure that this works?